3p/bodymovinanimation.js

```diff
@@ -20,6 +20,15 @@ import {loadScript} from './3p';
 import {parseJson} from '../src/json';
 import {setStyles} from '../src/style';
 
+const libSourceUrl = {
+  'canvas':
+    'https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.7.6/lottie_canvas.min.js',
+  'html':
+    'https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.7.6/lottie_html.min.js',
+  'svg':
+    'https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.7.6/lottie_svg.min.js',
+};
```

Comment:
I don't like the duplication here, ideally I'd define it in one place - but I thought you might care about separation of `3p` from `extensions` more than about duplication, so I left it as-is. Please let me know what your preference is.

Reply:
No need to share or repeat the object, see comment below.